crates/oxc_parser/src/js/expression.rs

```diff
@@ -202,13 +202,14 @@ impl<'a> Parser<'a> {
         let mut expressions = list.elements;
         let paren_span = self.end_span(span);
 
+        if expressions.is_empty() {
```

Comment:
why did you move this diagnostic up?

Reply:
Because if the expression is empty we need to return the diagnostic. So we don't neet to check `expression.len() == 1`